ee/codegen/src/context/node-context/inline-prompt-node.ts

```diff
@@ -64,21 +64,31 @@ export class InlinePromptNodeContext extends BaseNodeContext<InlinePromptNodeTyp
     const promptVersionData =
```

Comment:
revert the changes we made to this file. Once we iterate on the test above, the actual failure should become more obvious